src/snowcli/cli/object/stage/commands.py

```diff
@@ -27,18 +27,11 @@
 @app.command("list")
 @with_output
 @global_options_with_connection
-def stage_list(
-    stage_name: str = typer.Argument(None, help="Name of the stage."), **options
-) -> CommandResult:
+def stage_list(stage_name: str = StageNameOption, **options) -> CommandResult:
```

Comment:
Can we change the name `StageNameOption` to `StageNameArgument`?

Reply:
Done